content/en/events/kcseu/_index.md

```diff
@@ -21,14 +21,12 @@ Similar to North America summit last year, the summit in Europe will be
 informal with minimal programming, but plenty of space to hang out, catch up,
 and meet new contributors.
 
-We are looking for SIG meetings, session leaders for unconference sessions, plus 
-one or more pre-planned plenary sessions.  If you have an idea for a session, please
-look for our thread on session ideas coming soon. If your SIG wants to use the 
-Contributor Summit to have an in-person work session
-or meeting, please [sign up](https://docs.google.com/spreadsheets/d/1LS1q4VJajZkNZODdMnJ4eH3XnTsE_4em6pjQgToHyfw/edit?usp=sharing).
+We are looking for SIG meetings, [session leaders for unconference sessions](https://github.com/kubernetes/community/issues/6633), plus 
```

Comment:
```suggestion
We are looking for [SIG meetings](https://docs.google.com/spreadsheets/d/1LS1q4VJajZkNZODdMnJ4eH3XnTsE_4em6pjQgToHyfw/edit?usp=sharing), [session topics and leaders for unconference sessions](https://github.com/kubernetes/community/issues/6633), plus 
```

or did you want to remove the SIG signup sheet on purpose?

Reply:
thanks, change made